static/app/components/searchQueryBuilder/index.spec.tsx

```diff
@@ -178,7 +182,13 @@ describe('SearchQueryBuilder', function () {
         screen.getByRole('combobox'),
         'some free text brow{ArrowDown}'
       );
+
+      // XXX(malwilley): SearchQueryBuilderInput updates state in the render
+      // function which causes an act warning despite using userEvent.click.
+      // Cannot find a way to avoid this warning.
+      jest.spyOn(console, 'error').mockImplementation(jest.fn());
```

Comment:
Updating the state in a useEffect makes this warning go away, but does cause other issues with having a stale value for a render cycle so I don't want to do that. I tried wrapping everything I could in act to no avail 😞 

Hoping I can eventually get to the bottom of this, but throwing in the towel for now